components/dialog/stories/dialog.stories.js

```diff
@@ -19,8 +21,27 @@ export default {
 			control: { type: "text" },
 		},
 		content: { table: { disable: true } },
-		isDismissable: {
-			name: "Dismissable",
+		size: {
+			name: "Size",
+			type: { name: "string", required: true },
+			table: {
+				type: { summary: "string" },
+				category: "Component",
+			},
+			options: ["small", "medium", "large", "fullscreen", "fullscreenTakeover"],
```

Comment:
I wasn't sure how to handle the sizing and the fullscreen variants. The components I've worked on so far are using that more standardized t-shirt classes (like `--sizeL`), but dialog/index.css is using classes like `--small`. 

When we migrate dialog to S2, is that the time we could refactor those class names?

Reply:
Setting a `size` arg here for the small, medium, and large makes sense. Since this differs from the standard size naming and options.

I'm assuming the naming of those hasn't been changed yet to avoid breaking things downstream. You could ahead and create a backlog Jira issue about renaming these and getting them in sync in the future. And you're right that things may be changing with S2.

_(Also see related discussion on "_So the fullscreen and fullscreenTakeover don't really behave appropriately in storybook_". Fullscreen and fullscreenTakeover are their own classes that are separate from sizing, so wouldn't be options here.)_

Reply:
Made a note to make a card.